### PR TITLE
remove incorrect line

### PR DESCRIPTION
**Pull request by JacobGros**
# Description
this PR removes an incorrect line found 
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/885 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [x] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csmoperator/drivers/powerflex.md [hunk 1]
```diff
@@ -173,5 +173,4 @@ For detailed PowerFlex installation procedure, see the _Dell PowerFlex Deploymen
 5.  [Verify the CSI Driver installation](../#verifying-the-driver-installation)
     
 **Note** : 
-   1. "Kubelet config dir path" is not yet configurable in case of Operator based driver installation.
-   2. Also, snapshotter and resizer sidecars are not optional to choose, it comes default with Driver installation. 
+   1. snapshotter and resizer sidecars are not optional to choose, it comes default with Driver installation.
```
Comment (sharont58) [inline content/docs/deployment/csmoperator/drivers/powerflex.md:176]: Snapshotter and resizer sidecars installed by default.

**Review by shanmydell — CHANGES_REQUESTED** [review]
Implement the review comments

**Comment by JacobGros** [comment]
> Implement the review comments

fixed wording

**Review by sharont58 — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by shanmydell — APPROVED** [review]

**Pull request merged**